axone/chain.json

```diff
@@ -0,0 +1,121 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "axone",
+    "status": "live",
+    "website": "http://axone.xyz/",
+    "network_type": "mainnet",
+    "pretty_name": "Axone",
+    "chain_type": "cosmos",
+    "chain_id": "axone-1",
+    "bech32_prefix": "axone",
+    "slip44": 118,
+    "daemon_name": "axoned",
+    "node_home": "$HOME/.axoned",
+    "key_algos": [
+        "secp256k1"
+    ],
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "uaxone",
+                "low_gas_price": 0.001,
+                "average_gas_price": 0.005,
+                "high_gas_price": 0.01
+            }
+        ]
+    },
+    "staking": {
+        "staking_tokens": [
+            {
+                "denom": "uaxone"
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/axone-protocol/axoned",
+        "recommended_version": "v12.0.0",
```

Comment:
Properties here under "codebase" pertaining to the current version are redundant (except for recommended_version), and are required in a "versions.json" file.